regression-test/build.gradle.kts

```diff
@@ -42,6 +42,8 @@ dependencies {
   implementation("com.squareup.okhttp3:okhttp-tls:${okhttpLegacyVersion}") {
     exclude("org.bouncycastle")
   }
+  implementation(files("lib/envoy-0.4.0.04282021.aar"))
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.0")
```

Comment:
Move `coroutines` into `Dependencies` ?

Reply:
Yep, but this is just a draft.  No plans to land in current form with a chunky aar file on the branch.